.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        name: PHPUnit Tests
+        strategy:
+            fail-fast: false
+            matrix:
+                php-versions: [ '7.2', '7.3', '7.4', '8.0', '8.1', 'nightly' ]
+                monolog-versions: [ '1.*', '2.*' ]
+                include:
+                    -   php-versions: '7.1'
+                    -   php-versions: '7.1'
+                        composer-flags: '--prefer-lowest'
+                    -   php-versions: '8.1'
+                        minimum-stability: 'dev'
+                    -   php-versions: 'nightly'
+                        minimum-stability: 'dev'
```

Comment:
```suggestion
                include:
                    -   php: '7.4'
                        deps: lowest
                        deprecations: max[self]=0
                    -   php: '8.0'
                        deps: highest
```

Reply:
@derrabus Done, was PHP 7.1 removed intentionally?

Reply:
Oh no, please re-add it. Sorry.

Reply:
Done